warehouse/ingest-configuration/src/main/resources/config/shard-stats-config.xml

```diff
@@ -65,4 +65,10 @@
     <value>24</value>
     <description>hyperlog value for sparse precision</description>
   </property>
+
+  <property>
+    <name>partitioner.default.delegate</name>
```

Comment:
This is duplicative with ingest-config.xml and I think that ingest jobs will load both.  That could lead to confusion if either of them changes in the future.  Perhaps the stats job could be changed to load the configs differently.

Reply:
Cool. We talked offline but I'll put the notes here for others.

We're going to separate out the shard-stats-config.xml into a subdirectory. That'll exclude it from normal ingest since ingest doesn't care about any of these properties. Then we'll update buildIndexStats.sh to load it's configs from the subdir, which for now, will just contain the one config file.